generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_http.provider.ts

```diff
@@ -33,9 +33,7 @@ import { LoginModalService } from '../../shared/login/login-modal.service';
     <%_ } _%>
 import { StateStorageService } from '../../shared/auth/state-storage.service';
 <%_ } _%>
-<%_ if (!skipServer) { _%>
 import { AuthExpiredInterceptor } from './auth-expired.interceptor';
```

Comment:
Not very sure of this, Probabaly its better to add an abstract `auth-expired.interceptor` and `AuthServerProvider` or to skip the code calling these form the login service when skipServer is enabled. I would prefer the second option

Reply:
Hi @deepu105 , thx for your reply. I may be missing something here, but do we still have authentication when `skipServer` is enabled. ? Because the form login is always generated when the `authenticationType ` is different `oauth2` and the `authServerProvider` is always used to perform the authentication.

If it is mandatory, I would go with the 1st option you proposed.